dockerfiles/Dockerfile.rocm

```diff
@@ -2,10 +2,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------
-# Dockerfile to run ONNXRuntime with MIGraphX integration
+# Dockerfile to run ONNXRuntime with ROCm integration
 #--------------------------------------------------------------------------
 
-FROM ubuntu:18.04
+FROM ubuntu:20.04
```

Comment:
>ubuntu:20.04 [](http://example.com/codeflow?start=5&length=12)

would it be easier to use "rocm/pytorch:rocm5.2.3_ubuntu20.04_py3.7_pytorch_1.12.1"?